lib/interpreter.ml

```diff
@@ -201,7 +206,7 @@ module Make
 
 (* Internal typing *)
     type typ =
-      | TEmpty | TEvent | TEvents | TPair | TRel | TClassRel
+      | TEmpty | TEvent | TEvents | TPair | TRel | TTransRel | TClassRel
```

Comment:
I am not so sure that implicitly transitive relations deserve their own type. After all, they are relations. Can't we rely on implicit conversions between two forms of event relations?